.taskcluster.yml

```diff
@@ -48,10 +48,7 @@ tasks:
             - "git clone --quiet ${repository} bugbot &&
               cd bugbot &&
               git -c advice.detachedHead=false checkout ${head_rev} &&
-              pip install --quiet -r requirements-test.txt &&
```

Comment:
Why do we need to convert this into `test.sh`?

Reply:
We don't need to, but personally I like a simple way to run the tests locally in a way that matches CI as closely as possible (and generally dislike tests that run in VCS hooks). It also makes it mildly easier to change CI systems in the future (e.g. if we wanted to run this in GH actions instead).